examples/ghost/ghost-aws.yaml

```diff
@@ -0,0 +1,43 @@
+name: ghost
+org_name: <<org_name::org_name::Your Organization Name>>
+providers:
+  aws:
+    region: <<region::aws_region::Your AWS Region>>
+    account_id: <<account_id::aws_account_id::Your AWS Account ID>>
+modules:
+  - type: base
+  - type: dns
+    domain: <<domain::string::The dns domain for your Ghost deployment>>
+    delegated: false
+  - type: k8s-cluster
+  - type: k8s-base
+  - type: mysql
+    name: mysql
+    safety: false
+  - type: k8s-service
+    env_vars:
+      - name: database__client
+        value: "mysql"
+      - name: url
+        value: "https://${{module.dns.domain}}/dashboard"
+      - name: NODE_ENV
+        value: "production node current/index.js"
+      - name: mail__transport
+        value: "SMTP"
+      - name: mail__options__service
+        value: "Gmail"
+      - name: mail__options__auth__user
+        value: ""
+    secrets:
+      - mail__options__auth__pass
```

Comment:
secrets are no longer required, if this needs to be set by the user they would run `opta secret update` for it.

Reply:
Yes, but would still leave this in as it clarifies what secret needs to be passed in.

Reply:
Yup. So they only need this if they want to use the email feature which is not necessary. And I just documented the steps to setup the email

Reply:
cool, I'll double check on this as well - updating a secret with a env file containing a service